musicbot/entry.py

```diff
@@ -207,14 +207,14 @@ def to_json(self):
 
     # noinspection PyShadowingBuiltins
     async def _really_download(self, *, hash=False):
-        print("[Download] Started:", self.url)
+        print("[Download] Started: '%s' [s%]" % (self.title, self.url)
```

Comment:
Missing a parenthesis ) at the end.

`print("[Download] Started: '%s' [s%]" % (self.title, self.url))`